menu.rb

```diff
@@ -0,0 +1,61 @@
+require './app'
```

Comment:
- [ ] Great job extracting some of the logic from the `main.rb` into a separate file. However, I noticed to there are still some area's for improvement in your code 😉 
- [ ] To completely adhere to the project requirements, i will suggest that you ensure that `App.rb` adhere's to the Single Responsibility Principle. You can refer to [this lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) as a reference 💯 
- [ ] Currently, the input handling is done directly within the methods of the App class. To adhere to the Single Responsibility Principle and improve modularity, you can extract the input handling logic into a separate class or module.

- Here are some specific methods where input handling can be encapsulated and separated :fire: :

### Methods related to creating a book:

- [ ] create_book: Move the input prompts for title, author, and ISBN into a separate method or class responsible for handling book creation. 👍 

### Methods related to creating a person:

- [ ] create_person: Move the input prompts for selecting a `student` or `teacher` into a separate method or class responsible for handling person creation. 👍 
- Please do same for the `create_teacher` and `create_student` methods 😉 

### Methods related to selecting a book or person:

- [ ] select_book: Move the input prompts for selecting a book into a separate method or class responsible for handling book selection. 👍 
- [ ] select_person: Move the input prompts for selecting a person into a separate method or class responsible for handling person selection. 👍 

- By encapsulating and separating the input handling logic, you can achieve better modularity and maintainability. 💯 
- The main idea is to have a clear separation of responsibilities, where the App class focuses on high-level application logic while delegating input handling to dedicated classes or modules. 😉 
- [ ] Since you didn't make changes to the `App.rb` file I can't point out all the areas that need to be improved on so I will suggest that you ensure that throughout `App.rb`, you ensure that all requests for `input and input validation` should've been placed in the `main.rb` file or `classes/methods` whose responsibilities are to request or validate input. 😉